cliv2/internal/cliv2/cliv2_test.go

```
@@ -427,7 +427,6 @@ func Test_clearCache(t *testing.T) {
 	// check if directories that need to exist still exist
 	assert.FileExists(t, currentVersion)
 	assert.FileExists(t, lockfile)
-	assert.DirExists(t, cli.GetTempDir())
```

Suggestion: While this fixes the failing test, it possible hides errors in the implementation. I would suggest to adapt the test setup a bit to make this test pass instead.

---

In line 400, create the config the following way.

```
engine := app.CreateAppEngine()
config := engine.GetConfiguration()
```

---

And of course restore the assertion.

---

Thank you, you are the best 💯